control/lane_departure_checker/include/autoware_lane_departure_checker/lane_departure_checker_node.hpp

```diff
@@ -117,7 +117,7 @@ class LaneDepartureCheckerNode : public rclcpp::Node
   // Core
   Input input_{};
   Output output_{};
-  std::unique_ptr<LaneDepartureChecker> lane_departure_checker_;
+  std::unique_ptr<LaneDepartureChecker> autoware_lane_departure_checker_;
```

Comment:
I would suggest not to rename the variable, but if it is difficult to rename, then you can ignore this comment.

Reply:
Thanks!
I agree! addressed in https://github.com/autowarefoundation/autoware.universe/pull/7325/commits/19a4af04270442a9f47df0c2f26a1ac12befc11a